extensions/amp-base-carousel/0.1/test-e2e/test-hidden-controls.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getNextArrow} from './helpers';
+
+describes.endtoend(
+  'AMP carousel arrows with hidden controls',
+  {
+    testUrl:
+      'http://localhost:8000/test/manual/amp-base-carousel/hidden-controls.amp.html',
+    experiments: ['amp-base-carousel'],
+    environments: ['single'],
+  },
+  async function(env) {
+    let controller;
+
+    beforeEach(async () => {
+      controller = env.controller;
+    });
+
+    it('should not go to the next slide', async () => {
+      const nextArrow = await getNextArrow(controller);
+
+      // The test driver should throw an error, since the element is not clickable.
+      await expect(() => controller.click(nextArrow)).to.throw;
```

Comment:
Shouldn't this be `throw()` not `throw`?

Reply:
Also is it possible to confirm the error message describes the non-clickable behavior?

Reply:
Good catch, this wasn't actually checking anything before. Turns out I can't use this pattern for a few reasons, I tried changing our `expect` wrapper to make it work, but it seems to be quite involved.

Added a check to make sure it is a click error, I think it should work on Puppeteer and Webdriver.